hack/00-osd-managed-cluster-config-integration.yaml.tmpl

```diff
@@ -2173,7 +2173,23 @@ objects:
       kind: ClusterRole
       metadata:
         name: backplane-srep-readers-cluster
-      rules: []
+      rules:
+      - apiGroups:
+        - '*'
+        resources:
+        - pods
+        - services
+        - daemonsets
+        - roles
+        - deployments
+        - replicasets
+        - statefulsets
+        - jobs
+        - cronjobs
```

Comment:
It looks like we already have the perms get/list at cluster scope except roles, could you remove others and update the 'apiGroups" to "rbac.authorization.k8s.io/v1"?

Reply:
@wanghaoran1988 Done